bom/pom.xml

```diff
@@ -324,6 +324,16 @@
                 <artifactId>quarkus-amazon-sqs-deployment</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkiverse.amazonservices</groupId>
+                <artifactId>quarkus-amazon-sqs-messaging-connector</artifactId>
```

Comment:
Should we try to keep the existing convention at quarkus core ? Smallrye-reactive-messaging-aws-sqs ?

Reply:
@ozangunalp Quarkus core extension ends with Connector ? If so, I would vote for `quarkus-messaging-amazon-sqs-connector` for artifactId and `Quarkus Messaging - Amazon Services - SQS Connector` as the extension name.
Seems to respect both Quarkus core naming convention and this repo ones

Reply:
In Quarkus core the artifact id is  `quarkus-messaging-*`, artifact name is `Quarkus - Messaging - * - Runtime` etc. and the extension name is `Messaging * Connector`.

We can name this extension `Messaging - Amazon SQS - Connector` and the artifact `quarkus-messaging-amazon-sqs`